app/views/directives/add-secret-to-application.html

```diff
@@ -69,6 +69,23 @@
               </div>
             </div>
           </div>
+          <div ng-if="ctrl.addType === 'volume' && ctrl.application.spec.template.spec.containers.length > 1"  class="volume-options">
+            <h4>Containers</h4>
+            <div ng-if="ctrl.attachAllContainers">
+              The volume will be mounted into all containers. You can
+              <a href="" ng-click="ctrl.attachAllContainers = false">select specific containers</a>
+              instead.
+            </div>
+            <div ng-if="!ctrl.attachAllContainers" class="form-group">
```

Comment:
@jeff-phillips-18 Is there a max-height on the dialog in case there are a lot of containers?

Reply:
nope, I suppose there ought to be. Thanks!

Reply:
Set the max height on the list of containers.